tree/dataframe/test/dataframe_snapshot.cxx

```diff
@@ -1455,4 +1455,67 @@ TEST(RDFSnapshotMore, ZeroOutputEntriesMT)
    gSystem->Unlink(fname);
 }
 
+// Test for custom basket size in Snapshot, ROOT - #17418
+TEST(RDFSnapshotMore, CustomBasketSize)
```

Comment:
The test is only testing one branch of type `float`, i.e. a scalar. We need at least another test for collection types